framework/build.xml

```diff
@@ -226,6 +226,11 @@
             <package name="org.checkerframework.common.*"/>
             <package name="org.checkerframework.checker.*"/>
 
+            <fileset dir="${checker.loc}/${src}">
```

Comment:
The javadoc target in checker/build.xml has the same modifications as the javadoc target in framework/build.xml.  Only javadoc.jar target was modified to build jdk8 sources when jdk7orlower is true.  framework/build.xml does not have a javadoc.jar target.